cmd/corectl/main.go

```diff
@@ -79,7 +79,10 @@ func configGenerator(db *sql.DB, args []string) {
 
 	var flags flag.FlagSet
 	maxIssuanceWindow := flags.Duration("w", 24*time.Hour, "the maximum issuance window `duration` for this generator")
-	isSigner := flags.Bool("s", false, "whether this core is a signer")
+	flagK := flags.String("k", "", "local `pubkey` for signing blocks")
+	flagHSMURL := flags.String("hsm-url", "", "hsm `url` for signing blocks (mockhsm if empty)")
```

Comment:
I think it's confusing to just use `-url` is because there are two url arguments: the hsm url (as a flag) and a pubkey url (as an arg) `"usage: corectl config-generator [flags] [quorum] [pubkey url]..."`

Open to suggestions